### PR TITLE
7.0 upgrade test

### PR DESCRIPTION
**Pull request by graphaelli**
adds `--apm-server-tee` option to enable a few hardcoded options for testing upgrades to 7.0

run `scripts/ci/7.0-upgrade.sh` to try it out.  run `./venv/bin/pytest tests/server/test_upgrade.py -vv --showlocals` to iterate quickly.

### CODE REVIEW

**Comment by graphaelli** [comment]
Let's be sure to track outstanding issues with this effort in https://github.com/elastic/apm-server/issues/1807 instead of here.

File: scripts/compose.py [hunk 1]
```diff
@@ -557,12 +565,29 @@ def render(self):
...
+                # always build 7.0
+                backend["build"] = {
+                    "args": {
+                        "apm_server_base_image": "docker.elastic.co/apm/apm-server:7.0.0-SNAPSHOT",
+                        "apm_server_branch": "master",
+                        "apm_server_repo": "https://github.com/elastic/apm-server.git"
+                    },
+                    "context": "docker/apm-server"
+                }
+                backend["labels"] = ["co.elatic.apm.stack-version=7.0.0"]
```
Comment (simitt) [inline scripts/compose.py]: type `elatic`

Comment (graphaelli) [reply scripts/compose.py]: ha yeah, this typo is all over the repo

**Comment by simitt** [comment]
fyi pushing [commit](https://github.com/elastic/apm-integration-testing/pull/277/commits/ce1c8b4a07856f6c683ba2c31032baa119a9db15) makes the current test script succeed, in combination with https://github.com/elastic/apm-server/pull/1870.

File: tests/server/test_upgrade.py [hunk 2]
```diff
@@ -323,12 +323,16 @@ def status_code = http.remove("status_code");
...
+    def span_id = ctx._source.span.remove("id");
+    if (hex_id != null) {
+      ctx._source.span.id = hex_id;
+    } else if (span_id != null){
+      ctx._source.span.id = span_id.toString();
     }
     def parent = ctx._source.span.remove("parent");
     if (parent != null && ctx._source.parent == null) {
-      ctx._source.parent = ["id": parent];
+      ctx._source.parent = ["id": parent.toString()];
```
Comment (graphaelli) [inline tests/server/test_upgrade.py]: good one

**Comment by kuisathaverat** [comment]
@graphaelli @simitt 
Do we want to include a new stage to run `scripts/ci/7.0-upgrade.sh` script? if so, Should it run on PRs, merges, or other cases?

**Comment by graphaelli** [comment]
I think it could just run periodically, daily seems right.

**Pull request merged**